blueprints/flexberry-core/files/.jscsrc

```diff
@@ -0,0 +1,15 @@
+{
+  "excludeFiles": ["tests/unit/models/*"],
```

Comment:
Please add  `"esnext": true` option above, because it uses for 1.x.x versions of `broccoli-jscs`.